wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -245,22 +254,33 @@ void compile() throws IOException {
       types.addAll(protoFile.types());
     }
 
+    ExecutorService executor = Executors.newCachedThreadPool();
+    List<Future<Void>> futures = new ArrayList<>(MAX_WRITE_CONCURRENCY);
+
+    if (javaOut != null) {
```

Comment:
We’ll probably later want to refactor this to support “pluggable outputs” . . . doing Kotlin + Java + Swift might get ridiculous

Reply:
but not yet!